scripts/us_census/geojsons_low_res/download.py

```diff
@@ -70,7 +70,7 @@ def __init__(self):
         dc.set_api_key('dev')
         self.geojsons = None
 
-    def download_data(self, place='country/USA'):
+    def download_data(self, place='country/USA', level=1):
```

Comment:
Github won't let me comment above this line. Please update "# TODO(fpernice-google): Support downloading more than just US states." from line 25.